inscriptions/ISic001324.xml

```diff
@@ -191,19 +196,13 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140818">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Κούιντος Σόσσις<lb n="2"/>καὶ Πετρονία<lb n="3"/>Σοσσία Χείᾳ<lb n="4"/>τῇ <unclear>θ</unclear>υγατρί,
-                    ζησάσῃ<lb n="5"/>ἔτη ∙ ιʹ ∙
-                    <expan>
-                        <abbr>ἡμέρα</abbr>
-                        <ex>ς</ex>
-                    </expan>
-                    ∙ κ∙βʹ.
-                    <lb n="6"/>
+                   <lb n="1"/><persName type="attested"><name type="praenomen">Κούιντος</name> <name type="cognomen">Σόσσις</name></persName> 
+                   <lb n="2"/>καὶ <persName type="attested"><name type="nomen">Πετρονία</name> 
+                   <lb n="3"/><name type="cognomen">Σοσσία</name></persName> <persName type="attested"><name type="idionym">Χείᾳ</name></persName> 
+                   <lb n="4"/><hi rend="ligature">τῇ</hi> <hi rend="ligature">θυ</hi>γατρί ζησάσῃ 
+                   <lb n="5"/>ἔ<hi rend="ligature">τη</hi><g type="interpunct"/><num value="10">ι</num><g type="interpunct"/>ἡμέρα<supplied reason="omitted">ς</supplied> <g type="interpunct"/> <num value="22">κ <g type="interpunct"/> β</num>
```

Comment:
spacing is missing in the first part of the line, and the g type needs changing to g ref - but otherwise, nicely done!